tests/index.test.js

```diff
@@ -30,3 +49,32 @@ describe('Test that the basic routes return dummy data', () => {
 
   test.todo('POST /user/report')
 })
+
+describe('Test various utility functions', () => {
```

Comment:
I think it would be useful to have these tests in a `utilities.test.js` file of their own... eventually. Perhaps when we add a second utility or if one of us is compelled in a future change to migrate them. It doesn't seem worth holding this PR up though. 😉